python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -23,7 +23,7 @@
 import math
 
 from lsst.pex.config import Config, Field, ConfigurableField
-from .algorithmsLib import SingleGaussianPsf, DoubleGaussianPsf
+from . import SingleGaussianPsf, DoubleGaussianPsf
```

Comment:
Thanks for the clarification. For the record: I like your way better as I think it better guards against circular imports.